### PR TITLE
3. Add report

### PR DESCRIPTION
**Pull request by veronicatofino**

### CODE REVIEW

**Comment by veronicatofino** [comment]
imageReport

File: package.json [hunk 1]
```diff
@@ -29,5 +29,8 @@
     "@types/node": "^14.14.20",
     "protractor": "^7.0.0",
     "typescript": "^4.1.3"
+  },
+  "dependencies": {
+    "jasmine-spec-reporter": "^6.0.0"
```
Comment (yesid-lopez) [inline package.json]: move as a dev dependency

File: protractor/local.config.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { Config, browser } from 'protractor';
-
+import { reporter } from './helpers/reporter';
 export const config: Config = {
   framework: 'jasmine',
   specs: [ '../test/google.spec.js' ],
   seleniumAddress: 'http://localhost:4444/wd/hub',
+
```
Comment (yesid-lopez) [inline protractor/local.config.ts]: delete this empty line

Comment (veronicatofino) [reply protractor/local.config.ts]: @leonleo997 Corrected

File: protractor/local.config.ts [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
+import { reporter } from './helpers/reporter';
 export const config: Config = {
   framework: 'jasmine',
   specs: [ '../test/google.spec.js' ],
   seleniumAddress: 'http://localhost:4444/wd/hub',
   onPrepare: () => {
       browser.ignoreSynchronization = true;
-  }
+      reporter();
+    }
```
Comment (yesid-lopez) [inline protractor/local.config.ts]: indent these lines

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**